text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,502 @@
+---
+rfc pr: [#347](https://github.com/aws/aws-cdk-rfcs/pull/347)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/66
+---
+
+# CDK Self Managed Stack Set Support
+As a CDK user, I would like to define stack set infrastructure through native CDK constructs and create/update self-managed stack sets through CDK deploy command.
+
+* **Original Author(s):** @linsona
+* **Tracking Issue**: #66
+* **API Bar Raiser**: @skinny85
+
+## Working Backwards
+
+### CHANGELOG
+- feat(core): New Stack Set core type 
+- feat(assembly): New stack set artifcat type and schema
+- feat(cli): ```cdk deploy``` collects and creates/updates stack set 
+- feat(lib): API Calls/Logic for create/update stack sets
+
+### README
+The ```cdk.StackSet``` construct in CDK defines AWS resources in a stack set and stack set configuration/deployment preferences. This is different from the ```cdk.CfnStackSet``` construct which creates a CFN stack that deploys a CFN stack set. The ```cdk.StackSet``` construct creates the stack set directly if it does not exist, or updates a stack set and existing stack set instances while monitoring the stack set operation.
```

Comment:
General comment on the entire document:

Please be cognizant of the line lengths in this file (it should be readable also in its plaintext form).
I like to break lines according to sentence boundaries - it makes any future diff to this file read better than if the newlines are strewn randomly across sentences.

So:

```suggestion
The `cdk.StackSet` construct in CDK defines AWS resources in a stack set and its configuration and deployment preferences.
This is different from the `cdk.CfnStackSet` construct which creates a CloudFormation stack that deploys a CloudFormation stack set.
The `cdk.StackSet` construct creates the stack set directly if it does not exist,
or updates a stack set and existing stack set instances if it does,
while monitoring the stack set operation.
```

Same comment everywhere in the document.

Reply:
Ah gotcha (my ide was text wrapping), will fix